spec/lib/survey_notify_service_spec.rb

```diff
@@ -47,7 +47,7 @@
         expect(error.message).to match(/Communication with notify service failed/)
         expect(error.cause).to be_a Notifications::Client::RequestError
         expect(error.cause.code).to eq '403'
-        expect(error.cause.message).to eq [{"error" => "Forbidden", "message" => "You are not allowed to do this"}]
+        expect(error.cause.message).to eq [{ "error" => "Forbidden", "message" => "You are not allowed to do this" }]
```

Comment:
This is a linting change. Strange that it was failing locally, but not the build + the application could still be deployed. @tijmenb might have some insight on this?

Reply:
On Jenkins, `govuk-lint-ruby` is not running against the `spec` directory which is where this violation was located.

Reply:
Would need to update this in the `Jenkinsfile` but can do in a separate PR.

Reply:
OIC, could be worth pulling this out into another PR..